polars/polars-lazy/src/frame/parquet.rs

```diff
@@ -41,6 +42,41 @@ impl LazyFrame {
         Ok(lf)
     }
 
+    fn concat_impl(lfs: Vec<LazyFrame>, args: ScanArgsParquet) -> Result<LazyFrame> {
+        concat(&lfs, args.rechunk)
+            .map_err(|_| PolarsError::ComputeError("no matching files found".into()))
+            .map(|mut lf| {
+                if let Some(n_rows) = args.n_rows {
+                    lf = lf.slice(0, n_rows as IdxSize)
+                };
+                if let Some(rc) = args.row_count {
+                    lf = lf.with_row_count(&rc.name, Some(rc.offset))
+                };
+                lf
+            })
+    }
+
+    /// Create a LazyFrame directly from a parquet scan.
+    #[cfg_attr(docsrs, doc(cfg(feature = "parquet")))]
+    pub fn scan_parquet_files(paths: Vec<String>, args: ScanArgsParquet) -> Result<Self> {
+        let uniques: AHashSet<String> = AHashSet::from_iter(paths.into_iter());
```

Comment:
I don't think we need to deduplicate. It is the responsibility of the caller. I'd argue it even be a correct query if they send duplicate files in order to append them.

Reply:
Ok. I agree, you're right. I'll remove the `AHashSet` line.